dashpay/src/main/kotlin/org/dashevo/dashpay/BlockchainIdentity.kt

```diff
@@ -757,16 +794,22 @@ class BlockchainIdentity {
     ) {
 
         val query = DocumentQuery.Builder()
-            .where(listOf("saltedDomainHash", "in", saltedDomainHashes.map { "5620${it.value.toHexString()}" })).build()
-        val preorderDocuments = platform.documents.get("dpns.preorder", query)
+            .where(
+                listOf("saltedDomainHash",
+                "in",
+                saltedDomainHashes.map {
+                    it.value
+                }
+                )
```

Comment:
It looks like this portion of the code can be formatted.

Reply:
https://github.com/dashevo/android-dashpay/pull/23/commits/a5778b8df4348cc4748bb1d86ed3939e02fbc73a - fixed